### PR TITLE
Feat: Setup new Maven Module for CleaningServiceDummy

### PR DESCRIPTION
**Pull request by rainer-exxcellent**
## Description

Create maven module for the Cleaning Service Dummy.
I used org.eclipse.tractusx.bpdm.cleaning as base package name because org.eclipse.tractusx.bpdm.cleaning-service-dummy seems a little bit long.
At the moment no persistence to a database is configured.

### Solves
#421

### CODE REVIEW

**Comment by martinfkaeser** [comment]
Do we need a DOCKER_NOTICE.md like in the other modules?

File: bpdm-cleaning-service-dummy/pom.xml [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
```
Comment (martinfkaeser) [inline bpdm-cleaning-service-dummy/pom.xml:127]: We'll probably need some more test dependencies (Testcontainers, Wiremock?), but we can add them when needed.

Comment (rainer-exxcellent) [reply bpdm-cleaning-service-dummy/pom.xml:127]: I did not want to insert too many unused dependencies

File: bpdm-cleaning-service-dummy/src/main/kotlin/com/catenax/bpdm/cleaning/Application.kt [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+package com.catenax.bpdm.cleaning
```
[the remark this answers is not in the document]
Comment (rainer-exxcellent) [reply bpdm-cleaning-service-dummy/src/main/kotlin/com/catenax/bpdm/cleaning/Application.kt:20]: I chnaged the package name to org.eclipse.tractusx.bpdm.cleaning

**Comment by martinfkaeser** [comment]
Looks good!

**Comment by fabiodmota** [comment]
LGTM

**Review by nicoprow — CHANGES_REQUESTED** [review]
Since this pull request introduces a new Docker notice file it should be referenced in the list of notice files in the README

I also think the commits can be merged into one

**Comment by rainer-exxcellent** [comment]
I linked the Docker notice into the README file.

**Review by nicoprow — APPROVED** [review]

**Pull request merged**